src/cosl/coordinated_workers/nginx.py

```diff
@@ -133,7 +757,7 @@ def reload(self) -> None:
     def configure_pebble_layer(self) -> None:
         """Configure pebble layer."""
         if self._container.can_connect():
-            new_config: str = self._config_getter()
+            new_config: str = self._config_getter(self.are_certificates_on_disk)
```

Comment:
```suggestion
            new_config: str = self._config_getter(tls=self.are_certificates_on_disk)
```
more explicit what it's doing